indexer/indexer_tests/index_test.cpp

```diff
@@ -28,9 +28,9 @@ namespace
 class IndexTest : public MwmSet::Observer
 {
 public:
-  IndexTest() { TEST(m_index.AddObserver(*this), ()); }
+  IndexTest() { TEST(m_dataSource.AddObserver(*this), ()); }
```

Comment:
s/IndexTest/DataSourceTest/

also the filename

Reply:
done